### DISCUSSION TITLE
Github doesn't recognize my university-issued email

### DISCUSSION THREAD

**Discussion by den1kon** [community/community#69768] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm trying to reverify my academic affiliation in Github Education. I've started using Github Education when I was at school and everything was fine, I remember email being fine and then sending a photo of my School ID. Now I went to the university and university-issued email isn't recognizable by GitHub. In FAQ, it says that I should further prove my affiliation by sending a photo of Uni ID, which I have and can send, but the option to send it isn't there now at the "Get your GitHub benefits". I've tried contacting github support, but they now dropped support for it - "Individual manual reviews and status updates are no longer provided.". What can I do? Who can I contact?

**Comment by mantasu** [comment]
Based on my experience, when you apply, you should be able to take a picture with your webcam of the student ID and that will be checked automatically, no need to choose anything. Also, double-check if your university email includes `ed` or `edu`.
